Yoast/javascript
# Passive refactor clean up

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Deletes obsolete files and changes `SentenceNew` class to `Sentence`. Also adds an additional check in `parse.js` to parse classes.

## Relevant technical choices:

* The additional check in `parse.js` to parse classes was added to accommodate the fact that we created `Sentence` and `Clause` class using ES6.
* The empty `premium-configuration` branch with the same name should be deleted after this PR is merged.

## Test instructions

This PR can be tested by following these steps:

* run `yarn test` and make sure that everything passes.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-794
